app/src/main/java/com/duckduckgo/app/location/ui/SiteLocationPermissionDialog.kt

```diff
@@ -105,6 +107,18 @@ class SiteLocationPermissionDialog : DialogFragment() {
         }
     }
 
+    private fun populateSubtitle(subtitle: TextView) {
+        arguments?.let { args ->
+            val originUrl = args.getString(KEY_REQUEST_ORIGIN)!!
```

Comment:
This is a nitpick, but some of the functions in this file use `!!` on the `KEY_REQUEST_ORIGIN` argument but `populateFavicon()` uses a `let`. So it might be a good idea to standardize it.

Additionally, the `originUrl` could be extracted to a non-nullable in `validateBundleArguments()` and then `!!` wouldn't need to be used each time.